server/src/pages/leave-screenshots/server.js

```diff
@@ -4,7 +4,7 @@ const reactrender = require("../../reactrender");
 const { Shot } = require("../../servershot");
 const mozlog = require("../../logging").mozlog("leave-screenshots");
 
-const csrfProtection = csrf({cookie: true});
+const csrfProtection = csrf({cookie: {httpOnly: true, secure: true, sameSite: 'lax', key: '__Host-csrf'}});
```

Comment:
Seems like we shouldn't repeat this. Can we make a `csrf` module (maybe confusing to call it that) that includes the `csrfProtection` object, and maybe a portion of that cookie handling?